common/djangoapps/util/views.py

```diff
@@ -299,10 +299,9 @@ def _record_feedback_in_zendesk(
     zendesk_tags = list(tags.values()) + ["LMS"]
 
     # Per edX support, we would like to be able to route feedback items by site via tagging
-    current_site_orgs = configuration_helpers.get_current_site_orgs()
-    if current_site_orgs:
-        for org in current_site_orgs:
-            zendesk_tags.append("whitelabel_{org}".format(org=org))
+    current_site_name = configuration_helpers.get_value("SITE_NAME")
+    if current_site_name:
+        zendesk_tags.append("whitelabel_{org}".format(org=current_site_name))
```

Comment:
Shouldn't we keep the tag format the same since that's probably what their macros are driving off of? I was trying to keep it as consistent as possible. Will change the variable though as that doesn't make sense.